From 23f6a309f594f1deec25d8d4ef55248527d13dd4 Mon Sep 17 00:00:00 2001
From: Jin Xue <jin.xue@intel.com>
Date: Wed, 21 Nov 2018 11:11:08 +0800
Subject: [PATCH 1/6] primary audio hal

always compile it and place in the right folder

Change-Id: I9d4fe8f7fcbf370672ae6c2460c5edb848338c5e
Tracked-On:
Signed-off-by: Jin Xue <jin.xue@intel.com>
Reviewed-on: 652494
Reviewed-by: Zhang, Hongyu <hongyu.zhang@intel.com>
Tested-by: Zhang, Hongyu <hongyu.zhang@intel.com>
Reviewed-on: 657225
---
 Android.mk | 9 +--------
 1 file changed, 1 insertion(+), 8 deletions(-)

diff --git a/Android.mk b/Android.mk
index 68c26bf..ccf2807 100644
--- a/Android.mk
+++ b/Android.mk
@@ -12,8 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ifeq ($(INTEL_AUDIO_HAL),audio)
-
 LOCAL_PATH := $(call my-dir)
 
 include $(CLEAR_VARS)
@@ -35,15 +33,10 @@ LOCAL_C_INCLUDES += \
 	$(call include-path-for, audio-route) \
 	$(call include-path-for, audio-effects)
 
-#Preferred paths for all vendor hals /vendor/lib/hw
-LOCAL_PROPRIETARY_MODULE := true
-
-LOCAL_MODULE := audio.primary.$(TARGET_BOARD_PLATFORM)
+LOCAL_MODULE := audio.primary.$(TARGET_PRODUCT)
 
 LOCAL_MODULE_RELATIVE_PATH := hw
 
 LOCAL_MODULE_TAGS := optional
 
 include $(BUILD_SHARED_LIBRARY)
-
-endif # INTEL_AUDIO_HAL
-- 
2.7.4

